packages/jest-config-testing-library/README.md

```diff
@@ -69,3 +69,5 @@ module.exports = compose(
     withNTL(),
 );
 ```
+
+⚠️ Make sure `withNTL` is composed only after `withReactNative` from `@moxy/jest-config-react-native`.
```

Comment:
Consider adding this warning to others as well.

Reply:
Will do. 😉

Reply:
Done ✅